Assets/Scripts/CommandRegisterer.cs

```diff
@@ -41,4 +56,53 @@ private void Reset(string[] args)
         }
         SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex);
     }
+
+    /// <summary>
+    /// Copies the GPS coordinates of the specified waypoint (if it exists) to the clipboard.
+    /// </summary>
+    private void GetWaypoint(string[] args)
+    {
+        if(args.Length == 0)
+        {
+            SimulatorConsole.WriteLine("argument required");
+            return;
+        }
+        else
+        {
+            GameObject waypoint = GameObject.Find("Waypoints/" + args[0]);
+            if(waypoint == null)
+            {
+                SimulatorConsole.WriteLine("Cannot find that waypoint");
+            }
+            else
+            {
+                double[] GPS = Utilities.metersToGPS(new double[] {waypoint.transform.position.z, waypoint.transform.position.x});
+                GUIUtility.systemCopyBuffer = GPS[0].ToString("0." + new string('#', 339)) + ", "
+                                            + GPS[1].ToString("0." + new string('#', 339));
+                SimulatorConsole.WriteLine("Copied " + waypoint.name + " to clipboard");
+            }
+        }
+    }
+
+    /// <summary>
+    /// Lists all of the waypoints and their GPS coordinates on the Simulator Console.
+    /// </summary>
+    private void ListWaypoints(string[] args)
+    {
+        if (args.Length != 0)
+        {
+            SimulatorConsole.WriteLine("waypoints takes no arguments");
+            return;
+        }
+        else
+        {
+            foreach (Transform child in GameObject.Find("Waypoints").transform)
+            {
+                double[] GPS = Utilities.metersToGPS(new double[] {child.transform.position.z, child.transform.position.x});
+                SimulatorConsole.WriteLine(child.name + " (" +
+                    GPS[0].ToString("0." + new string('#', 339)) + ", " +
+                    GPS[1].ToString("0." + new string('#', 339)) + ")");
```

Comment:
Same as before, no need for these many digits. Also, could we explicitly label which one is lat and which one is lon?